setup.py

```diff
@@ -0,0 +1,31 @@
+from setuptools import setup, find_namespace_packages
+import os
+from pathlib import Path
+
+here = Path(__file__).parent.resolve()
+
+requirements = (here / "requirements.txt").read_text(encoding="utf8")
+long_description = (here / 'README.md').read_text(encoding='utf-8')
+
+VERSION = os.getenv('VERSION')
```

Comment:
you can't do it like this, it may work on your machine but not correctly. this file is processed when installing, that's why in the ydata lib, i write it into a file and then read that file when the setup runs.